edx_proctoring/views.py

```diff
@@ -756,16 +761,130 @@ def get(self, request, course_id):
 
         data_page = request.GET.get('page', 1)
         text_search = request.GET.get('text_search')
-        statuses_filter = request.GET.get('statuses')
+        status_filters = request.GET.get('statuses')
 
         enrollments = get_enrollments_can_take_proctored_exams(course_id, text_search)
 
+        # add custom attribute to better track performance of this endpoint against the
+        # number of proctoring eligible enrollments in the given course
+        set_custom_attribute('num_proctoring_eligible_enrollments', len(enrollments))
```

Comment:
@MichaelRoytman: This looks fine. Can you point to the doc that mentioned you should get a review from our team? I don't think these require any special review. Or, did you have a specific question? Thanks!

Reply:
@robrap Thank you for the review. I was referencing [this](https://openedx.atlassian.net/wiki/spaces/ENG/pages/54362736/Custom+Attributes+in+New+Relic) document.

> We only capture a few things now, but developers should feel free to add more (please just tag someone on the architecture team on your PR).

Reply:
Thanks. I’ll follow up with @ormsbee to see if we can remove that message or make it more clear when and why review might be required, or things to keep in mind.